ToneMapper/TMGLViewController.m

```diff
@@ -68,16 +50,24 @@ - (void)viewDidAppear:(BOOL)animated {
   if (!self.glkView) {
```

Comment:
Since this should occur only once in the lifecycle of the VC this initialization should be performed in <code>viewDidLoad</code> and not in <code>viewDidAppear:</code>.